Makefile

```diff
@@ -1,6 +1,16 @@
 # Config
 WORKERS = 4
 SHELL = /bin/bash
+MANAGE := python ./manage.py
```

Comment:
Oh, I missed your comment on this:

>Line 5: honcho run ./manage.py is not reliable when manage.py is not executable, which can happen when using Vagrant on Windows. Using the full honcho run python ./manage.py everywhere avoids this issue.

Sounds good then. Maybe add a comment in the Makefile to ensure people understand why this is necessary?